packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  Component,
+  Element,
+  State,
+  Event,
+  EventEmitter,
+  Method,
+  h,
+} from '@stencil/core';
+
+@Component({
+  tag: 'plmg-tabs',
+  styleUrl: 'plmg-tabs.scss',
+  shadow: true,
+})
+export class Tabs {
+  /**
+   * Reference to host HTML element.
+   */
+  @Element()
+  el: HTMLElement;
+
+  /**
+   * Store the tabs to be rendered in an array.
+   */
+  @State()
+  tabs: HTMLPlmgTabElement[] = [];
+
+  /**
+   * Event tabChange is emitted for onChange events when switching tabs.
+   */
+  @Event({ eventName: 'tabChange' })
+  onChange: EventEmitter;
+
+  /**
+   * On componentWillLoad dynamically grab tabs from the component to query and render.
+   */
+  componentWillLoad() {
+    this.tabs = Array.from(this.el.querySelectorAll('plmg-tab'));
+    if (this.tabs.length === 0) {
+      throw new Error('<plmg-tabs> Must have at least one tab');
+    }
+  }
+
+  /**
+   * Invoke this method on tab change to update active state and emit onChange handler.
+   */
+  @Method()
+  async openTab(index: number) {
+    if (index >= this.tabs.length) {
+      throw new Error(
+        `<plmg-tabs> Index ${index} is out of bounds of tabs length`
+      );
+    }
+    if (!this.tabs[index].disabled) {
+      this.tabs = this.tabs.map((tab, i) => {
+        tab.active = i === index;
+        return tab;
+      });
+      this.onChange.emit({ tabId: index });
+    }
+  }
+
```

Comment:
This may not follow the standard for keyboard navigation between tabs, from what I've just read you should be able to focus the tab list (focusing the active element) with a `tab` keypress, and then in our case, [left and right arrow presses should navigate to/from each tab](https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/). Currently, we have to use the tab key again to access the different tabs. May just want some tab-index magic, shown in the Examples in the link above